packages/ra-ui-materialui/src/input/SelectInput.tsx

```diff
@@ -161,6 +162,16 @@ export const SelectInput = (props: SelectInputProps) => {
         source: sourceProp,
     });
 
+    warning(
```

Comment:
The component should throw if it cannot render, not warn

Reply:
Agree, I'll change this also for the components from where I found this snippet.